src/components/FlyoutMenu/FlyoutMenu.test.jsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { fireEvent, render, waitFor, waitForElement } from '@testing-library/react';
+
+import FlyoutMenu, { FlyoutMenuDirection } from './FlyoutMenu';
+
+const applyButtonText = 'Apply';
```

Comment:
I think we can make the test less brittle by using the actual default values, e.g. `const applyButtonText = FlyoutMenu.defaultProps.i18n.applyButtonText`